components/sections/ExportTranslations.tsx

```diff
@@ -75,17 +72,28 @@ const ExportTranslations = ({ translations, categoryName, triggerLabel }: Export
 
   const translSep = translationSeparator === TRANS_SEP_CUSTOM ? customTranslationSeparator : translationSeparator;
   const phraseSep = phraseSeparator === PHRASE_SEP_CUSTOM ? customPhraseSeparator : phraseSeparator;
-  const phrases = translations
-    .map(
-      (translation) =>
-        translation.getTranslation(currentLanguage) +
-        (includeTranscriptions ? ` [${translation.getTranscription(currentLanguage)}]` : '') +
-        translSep +
-        translation.getTranslation(otherLanguage) +
-        (includeTranscriptions ? ` [${translation.getTranscription(otherLanguage)}]` : '') +
-        phraseSep
-    )
-    .map((translation) => unescapeTabsAndNewlines(translation));
+  const source =
+    categoryName != 'všechny fráze'
```

Comment:
use !== instead != ; but this line is wrong anyway;
!!!  Translations won't export when switched to UA . Because category name changes when language is switched. Category translations export won't export either. Test for both languages next time.

Quick solution for all translations:
I would omit category name when I want to export all translations.

Solution for category export:
I would use category ID instead of name to filter category.

Reply:
I have an idea about exportTranslations interface, what about to pass an array of category Ids. I could be useful in future if you decide to export only selected categories. For one category pass one Id like ['categoryIdHash'] . For all categories pass all Ids.